### DISCUSSION TITLE
Support for RustRover

### DISCUSSION THREAD

**Discussion by maciejlewandowskii** [GoogleCloudPlatform/cloud-code-intellij#3201] [Ideas]
RustRover is now in stable release, so it would be nice if you add support for it.

**Comment by ivanporty** [comment]
Thanks for request @maciejlewandowskii. Do you mean Cloud Code support, Gemini Code Assist support, both, and which is more important to you?

**Reply by jerryan999** [reply]
Gemini Code Assist support is what we need most
